### ISSUE TITLE
People movemement

### ISSUE THREAD

**Issue by agrignard** [CityScope/CS_Proxymix#26]
Right now there is 3 ways to make people move that goes in an increasing level of realism

1. Simple Goto (quick but not realist at all) as people are crossing through the wall
2. Go to on the grid (more realistic but seems very time consuming, and still people are using the same path
3. Using the pedestrian model which in theory should be the more realistic.

Should we integrate the 3 movements in the same model? 

@ptaillandier do you see any drawback to integrate everything and then having an easy way to switch between the different modes. It's ok if the model now requires to have the pedestrian plugins installed as soon as it's documented (I'll update the wiki for that)

**Comment by agrignard** [comment]
@ptaillandier do you think it's possible to integrate this in the same model to avoid to duplicate codes? or using an import in the pedestrain one for people that won't install the plugin and just want to run the simple one?

**Comment by ptaillandier** [comment]
Yep, that exactly what we are going to discuss tomorrow morning.

Le lun. 25 mai 2020 à 11:55, Arnaud Grignard <notifications@github.com> a
écrit :
